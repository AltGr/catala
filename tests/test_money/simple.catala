@Article@

/*
new scope A:
  param x content amount
  param y content amount 
  param z content amount

scope A:
  def x := $123.54
  def y := $8548,650.96
  def z := $250,000,000 *$ ((x /$ y) *. 0.2 %)
*/