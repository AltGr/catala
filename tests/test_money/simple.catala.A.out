[RESULT] x -> $123.54
[RESULT] y -> $8548650.96
[RESULT] z -> $7.23
